src/ray/raylet_client/raylet_client.cc

```diff
@@ -185,14 +185,6 @@ Status raylet::RayletClient::AnnounceWorkerPort(int port) {
   return conn_->WriteMessage(MessageType::AnnounceWorkerPort, &fbb);
 }
 
-Status raylet::RayletClient::SubmitTask(const TaskSpecification &task_spec) {
```

Comment:
Is it unused?

Reply:
Yea, seems unused. I removed it and it compiled.